BUILD.md

```diff
@@ -67,90 +76,86 @@ sudo pip3 install conan==1.59 grpcio-tools==1.57.0
 
 ```bash
-cd build
-cmake -G Ninja -DCMAKE_BUILD_TYPE=Release \
-  -DCMAKE_TOOLCHAIN_FILE=../ydb/clang.toolchain \
-  ../ydb
+git checkout 23.2.12
 ```
 
+### Check out the latest stable YDB CLI version for build
 
-### (optional) Configure with Ccache
+To build a latest stable version of a YDB CLI, check out the latest stable CLI tag from the repository. To do so, visit the https://github.com/ydb-platform/ydb/releases page, scroll down to the top-most 'YDB CLI' release, and use the provided 'tag' value in the `git checkout <tag>` command.
 
-With enabled Ccache, you can finish the compilation of all targets on supported Linux distributions in a few minutes. 
-  Or just `ydbd` or `ydb` binary in a couple of seconds. To optionally enable `Ccache` and enhance the compilation speed, follow these steps:
+At the moment of this documentation update, the latest YDB CLI release was [YDB CLI 2.5.0](https://github.com/ydb-platform/ydb/releases/tag/CLI_2.5.0) with a tag 'CLI_2.5.0', so the command looks like that:
 
-1. Install `Ccache` into `/usr/local/bin/` (We are using version `4.8.1`, you can use any version greater than `4.7`)
-    ```bash
-    (V=4.8.1; curl -L https://github.com/ccache/ccache/releases/download/v${V}/ccache-${V}-linux-x86_64.tar.xz | \
-     sudo tar -xJ -C /usr/local/bin/ --strip-components=1 --no-same-owner ccache-${V}-linux-x86_64/ccache)
-
-    ```
+```bash
+git checkout CLI_2.5.0
+```
 
-2. Configure `Ccache` to use remote storage using environment variables
-    ```bash
-    export CCACHE_REMOTE_STORAGE="http://cachesrv.ydb.tech:8080|read-only|layout=bazel"
-    export CCACHE_SLOPPINESS=locale
-    export CCACHE_BASEDIR=~/ydbwork/
-   
-    ```
-    <details>
-    <summary>or using Ccache config file</summary>
+## Configure
 
-    ```bash
-    ccache -o remote_storage="http://cachesrv.ydb.tech:8080|read-only|layout=bazel"
-    ccache -o sloppiness=locale 
-    ccache -o base_dir=~/ydbwork/
-   
-    ```
-    </details>
-3. Also, you should change Conan's home folder to the build folder for better cache hit 
+1. Change Conan's home folder to the build folder for better remote cache hit 
     ```bash
     export CONAN_USER_HOME=~/ydbwork/build
     ```
 
-4. Generate build configuration using `ccache`
+2. Generate build configuration using `ccache`
     ```bash
     cd build
     cmake -G Ninja -DCMAKE_BUILD_TYPE=Release \
     -DCCACHE_PATH=/usr/local/bin/ccache \
     -DCMAKE_TOOLCHAIN_FILE=../ydb/clang.toolchain \
-    ../ydb
-   
+    ../ydb  
     ```
 
-## Build
+## Build 
 
-To build all binary artifacts (server YDBD, client YDB, unittest binaries) run:
+### Build YDB Server
+
+To build YDB Server run:
 ```bash
-ninja
+ninja ydb/apps/ydbd/all
 ```
 
 A YDB server binary can be found at:
 ```
 ydb/apps/ydbd/ydbd
 ```
 
-## Build and Test YDB CLI
+### Build YDB CLI
 
-To build YDB CLI (ydb):
+To build YDB CLI (Command Line Interface utility) run:
```

Comment:
it is better to explain what CLI stands for in the beginning when we first mention it, not here almost at the end

Reply:
It is actually there, in the first sentence.

Reply:
Then this is a duplicate explantation and thus not needed?